### PR TITLE
#165020163 User should be able to reset password

### PR DESCRIPTION
**Pull request by niyobobo**
### What does this PR do?
Give a user functionality to change Password
### Description of Task to be completed?
- User should request to reset password
- User should receive an email with a link to reset password
- User should reset his/her password
### How should this be manually tested?
- Clone the repository locally with `git clone https://github.com/andela/ah-kgl-avengers-frontend.git` command.
- Check out to this branch by running `git checkout ft-user-reset-password-165020163`.
- Install dependencies by running `npm install`.
- Update the .env file according to the .env.sample file
- Start the server with `npm start` and put this url `http://localhost:8000/reset` in the address bar of your browser then follow the instruction of resetting your password.
### Any background context you want to provide?
- User should have an account on `AuthorsHaven` to be able to reset an account.
### What are the relevant pivotal tracker stories?
[#165020163](https://www.pivotaltracker.com/n/projects/2321840/stories/165020163)
### Screenshots (if appropriate)
image
image
image

### Questions:
N/A

### CODE REVIEW

File: src/redux/action-types/resetPassword.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const REQUEST_RESET = 'REQUEST_RESET';
+export const REQUEST_RESET_SENT = 'REQUEST_RESET_SENT';
+export const REQUEST_RESET_FAIL = 'REQUEST_RESET_FAIL';
+export const OPEN_RESET = 'OPEN_RESET';
```
Comment (shemaeric) [inline src/redux/action-types/resetPassword.js]: @niyobobo I think this action type is not used.

File: src/redux/action-types/resetPassword.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+export const SET_PASSWORD = 'SET_PASSWORD';
```
Comment (shemaeric) [inline src/redux/action-types/resetPassword.js]: same here

**Review by shemaeric — CHANGES_REQUESTED** [review]
Good Work @niyobobo , you can consider small changes requested and also I am still thinking that the words ` No problems, it happens! Just enter the email address.
          We will send instructions on how to reset your password.`on the page are much and could be better if reduced to a single clear word.

**Comment by ezkemboi** [comment]
Also, improve the test coverage for the file changes for resetting password and the feedback received in the stand-up meeting.

**Review by ezkemboi — APPROVED** [review]
Good work so far. But please add tests for reducers and action creators. 
Generally good work.

File: src/redux/action-types/resetPassword.js (hunk 1 above)
Comment (niyobobo) [reply src/redux/action-types/resetPassword.js]: Solved it

File: src/redux/action-types/resetPassword.js (hunk 2 above)
Comment (niyobobo) [reply src/redux/action-types/resetPassword.js]: Solved it

**Pull request merged**